rebasehelper/tests/public_api/test_specfile.py

```diff
@@ -0,0 +1,73 @@
+# -*- coding: utf-8 -*-
+#
+# This tool helps you rebase your package to the latest version
+# Copyright (C) 2013-2019 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, write to the Free Software Foundation, Inc.,
+# 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+#
+# Authors: Petr Hráček <phracek@redhat.com>
+#          Tomáš Hozza <thozza@redhat.com>
+#          Nikola Forró <nforro@redhat.com>
+#          František Nečas <fifinecas@seznam.cz>
+
+import os
+import shutil
+
+import pytest  # type: ignore
+
+from rebasehelper.specfile import SpecFile
+from rebasehelper.tests.conftest import SPEC_FILE, TEST_FILES_DIR
+
+
+@pytest.mark.public_api
+class TestSpecFile:
+    def test_constructor(self, workdir):
+        shutil.copy(os.path.join(TEST_FILES_DIR, SPEC_FILE), workdir)
+        # check that no mandatory argument was added
+        assert isinstance(SpecFile(SPEC_FILE), SpecFile)
+        # check that arguments were not renamed or removed
+        assert isinstance(SpecFile(path=SPEC_FILE, sources_location=workdir, predefined_macros=None), SpecFile)
+
+    def test_update_changelog(self, spec_object):
+        assert not spec_object.update_changelog('test2')
```

Comment:
`not` doesn't assert that the value is `None`, it could be for example `[]` or `''` or even `0`.